CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to Noir
+
+Thank you for your interest in contributing to Noir! There are many ways to contribute
+and we appreciate all of them.
+
+## Getting help
+
+There are many ways you can get help when you're stuck. The best way to get started is by asking for help in the #noir
+channel on the [Aztec discord](https://discord.gg/aztec). 
+
+## Submitting Issues
+
+Submitting an issue in this repository is also a great way to contribute and get help with problems that you run into.
+When submitting an issue, please include as much detail as possible about the error message, your environment (OS, version numbers, etc) and the 
+steps to reproduce the error.
+
+## Pull Requests
+
+Noir is still under active development. If you would like to submit a PR to the repo, please reach out to the team on Discord to discuss
+ahead of time.
```

Comment:
I think this is too burdensome a requirement. If someone wants to submit a PR they can just submit one and shouldn't need permission first. It is only if they want to contribute and aren't sure what to work on or are stuck that they would benefit from contacting us through discord

Reply:
ok that makes sense. ill remove this section entirely and we can add something later as needed.